go/worker/compute/executor/committee/node.go

```diff
@@ -150,8 +150,9 @@ type Node struct { // nolint: maligned
 	// Guarded by .commonNode.CrossNode.
 	proposingTimeout bool
 
-	commonNode *committee.Node
+	prevEpochInCommittee bool
```

Comment:
Should probably also add the notice that the flag is guarded by `.commonNode.CrossNode` (or just move it to the group above).

Reply:
done :+1: